app/src/androidTest/java/org/kiwix/kiwixmobile/page/history/HistoryRobot.kt

```diff
@@ -28,9 +27,6 @@ import org.kiwix.kiwixmobile.R
 fun history(func: HistoryRobot.() -> Unit) = HistoryRobot().applyWithViewHierarchyPrinting(func)
 
 class HistoryRobot : BaseRobot() {
-  init {
-    assertDisplayed(R.string.history_from_current_book)
```

Comment:
hi @gouri-panda,

Why we are removing init block  ?

Reply:
Often times `BaseRobot` class doesn't instantiate before it's checking the `assertDisplayed()` function.